internal/bminventory/inventory.go

```diff
@@ -918,6 +918,29 @@ func (c clusterInstaller) install(tx *gorm.DB) error {
 	return nil
 }
 
+func (b *bareMetalInventory) storeOpenshiftClusterID(ctx context.Context, clusterID string) error {
+	reader, _, err := b.objectHandler.Download(ctx, fmt.Sprintf("%s/%s", clusterID, "bootstrap.ign"))
```

Comment:
Please add logs for every failing condition so it would be easier to understand in case of a failure.